### PR TITLE
refactor: migrate AddParticipants to React

### PR DESCRIPTION
**Pull request by przemvs**
Panel switch functionality will be prepared on end of migration all panel elements, because we need to write additional functionality to be distinguished migrated and not migrated elements.

image
image

### CODE REVIEW

File: src/script/page/RightSidebar/AddParticipants/AddParticipants.tsx [hunk 1]
```diff
@@ -0,0 +1,334 @@
...
+    const userEntities = getUserEntities();
+    setContacts(userEntities);
+  }, []);
+
+  return (
+    <div id="add-participants" className="add-participants panel__page panel__page--visible">
+      <PanelHeader
+        showBackArrow
+        goBackUie="go-back-add-participants"
+        onGoBack={onBack}
```
Comment (arjita-mitra) [inline src/script/page/RightSidebar/AddParticipants/AddParticipants.tsx:189]: Please talk to Astrid and confirm if you need to add these props for PanelHeader goBackTitle, title, closeBtnTitle

**Review by arjita-mitra — CHANGES_REQUESTED** [review]

File: src/script/page/RightSidebar/AddParticipants/AddParticipants.tsx (hunk 1 above)
Comment (przemvs) [reply src/script/page/RightSidebar/AddParticipants/AddParticipants.tsx:189]: Don't worry about it. It's prepared :D 

image
image

File: src/script/page/RightSidebar/AddParticipants/AddParticipants.tsx [hunk 2]
```diff
@@ -0,0 +1,334 @@
...
+    await addUsers();
+    onBack();
+  };
+
+  const onSearchInput = async (value: string) => {
+    await searchServices(value);
+    setSearchInput(value);
+  };
+
+  const getUserEntities = useCallback(() => {
```
Comment (PatrykBuniX) [inline src/script/page/RightSidebar/AddParticipants/AddParticipants.tsx]: Is this function necessary here? We could just use `useMemo`, so there is no need to call this function on initial render (line 187) - we lose 1 rerender.

Comment (przemvs) [reply src/script/page/RightSidebar/AddParticipants/AddParticipants.tsx]: Ahh.. sure, you're right. I'll fix it :)

Comment (przemvs) [reply src/script/page/RightSidebar/AddParticipants/AddParticipants.tsx]: done.

**Review by PatrykBuniX — APPROVED** [review]
Looks fine 👍

**Pull request merged**